WcaOnRails/lib/url_validator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# lib/file_size_validator.rb
+class UrlValidator < ActiveModel::EachValidator
+  URL_RE = %r{\Ahttps?://\S+\z}
+  VALID_URL_MESSAGE = "must be a valid url starting with http:// or https://"
+
+  def validate_each(record, attribute, value)
+    if value.present? && !URL_RE.match(value)
```

Comment:
As far as I can see it doesn't validate presence, shouldn't we include presence validation in the relevant models then? (Or do I just miss the point?)

Reply:
Makes perfect sense!